app/decorators/book_decorator.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class BookDecorator < Draper::Decorator
+  delegate_all
```

Comment:
[Note]
It will delegate all of the methods in book model to the decorator => you can call the methods from the decorator object like the model object.